front/src/config/i18n/en.json

```diff
@@ -276,6 +277,12 @@
         "no": "No",
         "preview": "Preview",
         "showPreviewButton": "Show preview"
+      },
+      "clock": {
+        "analog": "Analog",
+        "digital": "Digital",
+        "date": "{{day}}, {{month}} {{dayNumber}}, {{year}}",
```

Comment:
Can't you use dayjs format according to user language instead?

Reply:
Yes, but we cannot have the day without the hours : https://day.js.org/docs/en/display/format#list-of-localized-formats

Reply:
You can format as you want: https://day.js.org/docs/en/display/format

Reply:
I don't find a good way to use dayjs directly because the `LocalizedFormat` proposed by this library don't have the good display. 
There is no way to add second on the time or have the day without the time. 
If you have any idea :)